lte/gateway/c/oai/tasks/mme_app/mme_app_itti_messaging.c

```diff
@@ -642,3 +645,44 @@ void mme_app_itti_sgsap_ue_activity_ind(
   }
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
+
+// Extract MCC and MNC from the imsi received and match with
+// configuration
+int mme_app_match_fed_mode_map(const uint8_t* imsi, imsi64_t imsi64) {
```

Comment:
Why do we need both imsi and imsi64 here? We can always derive one from the other. 

Also, can we re-use the `match_fed_mode_map` defined in https://github.com/magma/magma/blob/master/lte/gateway/c/oai/lib/s6a_proxy/S6aClient.cpp#L105 . It will be better to move that out to c/oai/common/